src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -5321,7 +5321,7 @@ protected virtual bool HitTest(Touch touch)
         /// <summary>
         /// Voice interaction name for voice touch.
```

Comment:
I was thinking about adding more information about Voice Touch, but maybe it's not the place for that. I would suggest only changing how it is written to: Voice Touch (with capital letters). It will look more like a specific term.

Reply:
Thank you for review.
I have changed as your comment.